plugins/modules/dcnm_network.py

```diff
@@ -551,6 +573,74 @@ def diff_for_attach_deploy(self, want_a, have_a, replace=False):
 
                         if want.get("isAttached") is not None:
                             if bool(have["isAttached"]) and bool(want["isAttached"]):
+                                torports_configured = False
+
+                                # Handle tor ports first if configured.
+                                if want.get("torports"):
```

Comment:
Quite a bit of nested logic here.  This is not something we should necessarily change now but just bringing it up for discussion.  Let's explore if we could improve the readability / long term maintenance by breaking this up into smaller helper functions.  What do you think?  Again, not a blocker for this PR.

Reply:
Sure, we can do that for future commits